tests/Agent/IntegrationTests/IntegrationTests/Api/AgentApiTests.cs

```diff
@@ -56,7 +56,7 @@ public void Test()
 
             var actualMetrics = _fixture.AgentLog.GetMetrics();
             var errorTrace = _fixture.AgentLog.GetErrorTraces().FirstOrDefault();
-            var errorEventPayload = _fixture.AgentLog.GetErrorEvents().FirstOrDefault();
+            var errorEventPayload = _fixture.AgentLog.GetErrorEventPayloads().FirstOrDefault();
```

Comment:
Optional, this test may benefit from just using `GetErrorEvents()` and comparing counts of that only.
Is this test specifically testing the `EventsSeen` value correctness or the number of errors recorded?